fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java

```diff
@@ -543,6 +544,23 @@ public void removeOldExportJobs() {
                     }
                 }
             }
+
+            int maximumHistoryJobNum = Config.maximum_history_job_num;
+            List<Map.Entry<Long, ExportJob>> jobList = new ArrayList<>(exportIdToJob.entrySet());
+            jobList.sort(Comparator.comparingLong(entry -> entry.getValue().getCreateTimeMs()));
```

Comment:
You can sort the jobs only when `exportIdToJob.size() > maximumHistoryJobNum`, to save some time

Reply:
done